docs/supported_docker_environment/windows.md

```diff
@@ -17,16 +17,12 @@ If you wish to use Testcontainers on Windows, please confirm that it works corre
 effort.
 
 ### MySQL containers
-* MySQL server prevents custom configuration file (ini-script) from being loaded due to security measures ([link to feature description](database_containers.md#using-an-init-script))
+* MySQL server prevents custom configuration file (ini-script) from being loaded due to security measures ([link to feature description](../modules/databases/index.md#using-an-init-script))
```

Comment:
is it still true, btw?

Reply:
🤔 I don't actually know... We might need to look at when this was added to the original docs.

Reply:
https://github.com/testcontainers/testcontainers-java/issues/1166
reported, let's keep it for now